integreat_cms/cms/templates/ajax_poi_form/poi_box.html

```diff
@@ -34,14 +41,16 @@
                    data-default-placeholder="{{ poi_title_placeholder }}" />
             <div class="absolute inset-y-0 right-0 flex items-center px-2 text-gray-800">
                 <button id="poi-remove"
-                        title="{% translate "Remove location from event" %}"
+                        title="{% translate "Remove location" %}"
                         {% if form.disabled %}disabled{% endif %}>
                     <i icon-name="trash-2" class="h-5 w-5"></i>
                 </button>
             </div>
         </div>
         <p class="text-sm italic block mt-2 mb-2">
-            {% translate "Create an event location or start typing the name of an existing location. Only published locations can be set as event venues" %}.
+            {% if current_menu_item == "events_form" %}
+                {{ help_text }}
+            {% endif %}
```

Comment:
Same here. I would either move this code to the event form, or if that does not look similar, maybe pass a help text variable to this component